src/data/customer/Order.js

```diff
@@ -633,4 +640,7 @@ export const mergeLocationDataWithErrors = (errorContents, location) => {
       return errorList;
     }, []);
   }
+
+  /* Response empty array if there's no error from dynamodb */
```

Comment:
You don't check if there is no error for example
```js
if(error != null) return error
return []
```
If you had written a unit test you would have fount out :D